all.json

```diff
@@ -11,6 +11,82 @@
 		"162.222.226.37",
 		"1776coin.org",
 		"1chaincrypto.com",
+		"24x7-bittreks.com",
+		"24x7-ledger.com",
+		"alienheroesworld.io",
```

Comment:
Not sure i see the "trying to scam" here, it does look like an NFT?

Reply:
"24x7-bittreks.com", = unborn bittrex.com scam (100% confidence)
"24x7-ledger.com", =  unborn ledger.com scam (100% confidence)